dev/spark-test-image-util/docs/build-docs

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
Can we document this in docs/README.md?

Reply:
Okay, let me do it as a follow up pr.

Reply:
Please review: https://github.com/apache/spark/pull/49013
thanks!